cpp/tests/io/parquet_test.cpp

```diff
@@ -1174,6 +1177,38 @@ TEST_F(ParquetWriterTest, DeviceWriteLargeishFile)
   auto custom_tbl = cudf_io::read_parquet(custom_args);
   CUDF_TEST_EXPECT_TABLES_EQUAL(custom_tbl.tbl->view(), expected->view());
 }
+
+TEST_F(ParquetWriterTest, PartitionedWrite)
```

Comment:
Not sure if this makes sense, but it's probably good to add partitioned write tests with empty table(s). This can be both no data and no columns.